torchvision/models/_api.py

```diff
@@ -203,19 +204,45 @@ def wrapper(fn: Callable[..., M]) -> Callable[..., M]:
     return wrapper
 
 
-def list_models(module: Optional[ModuleType] = None) -> List[str]:
+def list_models(
+    module: Optional[ModuleType] = None,
+    include_filters: Union[Iterable[str], str, None] = None,
+    exclude_filters: Union[Iterable[str], str, None] = None,
```

Comment:
Some name bikeshedding: should we just name these `include` and `exclude`?

```py
list_models(include="resnet*")
```

looks just as clear as 

```py
list_models(include_filters="resnet*")
```

IMHO.